packages/components/organisms/f-checkout/test/specs/component/f-checkout-collection.component.spec.js

```diff
@@ -1,38 +1,35 @@
 import forEach from 'mocha-each';
-import DemoControls from '../../../test-utils/component-objects/demo-controls';
 import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.component';
 
 describe('f-checkout "collection" component tests', () => {
-      beforeEach(() => {
+    before(() => {
```

Comment:
Is this change to run the set up once only intentional? Or should the before block re-run before each IT statement executes?